applications/acoustic_conservation_equations/vibrating_membrane/tests/skew_symmetric.output

```diff
@@ -98,7 +98,7 @@ Calculate error for pressure at time t = 2.5300e-02:
   Absolute error (L2-norm): 1.56490e-02
 
 Calculate error for velocity at time t = 2.5300e-02:
-  Absolute error (L2-norm): 9.44139e-03
+  Absolute error (L2-norm): 2.83242e-02
```

Comment:
The absolute error is exactly scaled by the density.

Reply:
did you think about using relative errors?

Reply:
Yes, sadly in this particular test case, the analytical solution of the pressure and velocity are zero at certain times and therefore, we should use absolute errors here.